examples/vols/ex.yml

```diff
@@ -0,0 +1,34 @@
+version: 0.1-dev
+
+services:
+- name: db
+  containers:
+  - image: mysql
+    mounts:
+    - volumeName: db
+      mountPath: /var/lib/mysql
+
+- name: backup
+  containers:
+  - image: backup
+    mounts:
+    - volumeName: db
+      mountPath: /app/store
+      volumeSubPath: foo/bar
```

Comment:
Total thumbsup for this feature, I do need it for my usecase:

https://github.com/brainstorm/cgtd/commit/344fa1a15d5874c3dff17e91537864b77b5fd2b1

Would it be reasonable to include support for paths like `$HOME` in the spec?